contribute/views.py

```diff
@@ -18,11 +18,12 @@ def get(self, request, *args, **kwargs):
             user_profile = None
 
         if user_profile:
-            user_language = UserLanguage.objects.get(user=user_profile)
-            if user_language.is_moderator:
-                user_moderator = True
-            else:
-                user_moderator = False
+            user_languages = UserLanguage.objects.filter(user=user_profile)
+            user_moderator = False
```

Comment:
You could also put this in the else clause of the for loop. Both is fine, I just wanted to tell you in case you didn't know :)

Reply:
thanks